pkg/registry/core/service/rest_test.go

```diff
@@ -672,6 +627,19 @@ func TestServiceRegistryResourceLocation(t *testing.T) {
 	}
 }
 
+func checkLocation(r rest.Redirector, ctx api.Context, id, e string, t *testing.T) {
+	location, _, err := r.ResourceLocation(ctx, id)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
```

Comment:
OK, I will add some information in error message. I think we could decide it by id field. Thanks for your review.